packages/bvaughn-architecture-demo/components/console/MessagesList.tsx

```diff
@@ -86,6 +88,8 @@ function MessagesList({ forwardedRef }: { forwardedRef: ForwardedRef<HTMLElement
     const loggableExecutionPoint = getLoggableExecutionPoint(loggable);
     const isLoaded =
       loadedRegions !== null && isPointInRegions(loggableExecutionPoint, loadedRegions.loaded);
+    const includesFirstConsoleNag = currentUserInfo?.nags?.includes(Nag.FIRST_CONSOLE_NAVIGATE);
```

Comment:
Something about the way we model this stuff irks me.

Maybe a small part of it is that we call these things "nags" – which is an inherently negative term.

I think the bigger part though is the way we expose them as an array of strings that we do an `includes` check on. Even if that's how we model things in GraphQL, I think we could expose it to our views with a better API (e.g. `currentUserInfo.onboarding.showInitialConsoleHoverPrompt`)

Reply:
Totally, I was going to copy over the utility we have right now for this so it's more explicit, but is probably worth investing whatever renaming we want to do now before this starts getting added to new code.